build.gradle

```diff
@@ -26,8 +26,8 @@ buildscript {
     classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.4'
     classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.0.0'
     classpath 'com.android.tools.build:gradle:1.5.0'
-    classpath 'com.redhat.ceylon.gradle:android:0.0.1-SNAPSHOT'
-//    classpath "com.athaydes.gradle.ceylon:ceylon-gradle-plugin:1.1"
+//    classpath 'com.redhat.ceylon.gradle:android:0.0.1-SNAPSHOT'
```

Comment:
> Do you have any idea about how to get this done?

Yes, the created modules have a `mapping.txt` which maps generated class files and their source ceylon files. You can use that. I think the ant task does incremental compilation that way. It doesn't give you dependencies, though, but it's a start. I bet we could generate that mapping too if you want.